docs/antrea-network-policy.md

```diff
@@ -267,8 +273,14 @@ be enforced in the order in which they are written.
 **egress**: Each ClusterNetworkPolicy may consist of zero or more ordered set
 of egress rules. Each rule, depending on the `action` field of the rule, allows
 or drops traffic which matches both the `to` and `ports` sections. 
-Also, each rule has an optional `name` field, which is unique within the policy 
-describing the intention of this rule.
+Also, each rule has an optional `name` field, which should be unique within 
+the policy describing the intention of this rule. If users didn't provide `name`,
+antrea will auto-generate it. The auto-generate name will be in format: 
```

Comment:
we could simply say something like 'the same rule name auto-generation process is applied to egress rules'